### PR TITLE
Mil mag fix

### LINKED ISSUE #452: Military magboots have no in-window toggle, and cannot be toggled with the "toggle-magboots" verb
https://github.com/Eclipse-Station/NEV-Northern-Light/blob/7559a6a86ffc8b82156299e1b1cc0c8f3bb61a09/code/modules/clothing/shoes/magboots.dm

/obj/item/clothing/shoes/magboots/merc

The only way to turn them off or on is by using them in hand.

### PR DESCRIPTION
**Pull request by MciReloaded**
## About The Pull Request

Allows the users of Military Magboots to toggle them on and off, as requested in Issue  #452
## Why It's Good For The Game

Why should they have to have their magboots on all the time?
## Changelog
:cl:
fix: Allows Mercenaries to toggle their magboots on and off.
/:cl:

EJC Edit: Closes #452

### CODE REVIEW

File: code/modules/clothing/shoes/magboots.dm [hunk 1]
```diff
@@ -58,8 +58,8 @@
 	species_restricted = null
 	force = WEAPON_FORCE_PAINFUL
 	overslot = FALSE
-	magpulse = TRUE
+	magpulse = 0
```
Comment (drexample) [inline code/modules/clothing/shoes/magboots.dm:61]: Do not use numbers for booleans

**Review by drexample — CHANGES_REQUESTED** [review]
Do not use numbers for booleans

**Comment by EvilJackCarver** [comment]
Can you please use the [magic words](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) so that the issue is linked and will close automatically if this is merged?

File: code/modules/clothing/shoes/magboots.dm (hunk 1 above)
Comment (MciReloaded) [reply code/modules/clothing/shoes/magboots.dm:61]: Hey it's what other things used but ye I can fix thaht

**Review by EvilJackCarver — APPROVED** [review]

**Comment by MciReloaded** [comment]
I mean, it fixes #452

**Pull request merged**